manifest.php

```diff
@@ -57,16 +60,74 @@
             RegisterNpmPaths::class,
             RegisterItemPreviewerRegistryService::class,
             RegisterAssetTreeBuilder::class,
+            [
```

Comment:
Would not be better to have an installation script to set it? Imagine that for every new class/action we will need to repeat it here.

Reply:
Do you mean to "wrap" this tool into installation script and call installation script instead of tool?
```
class BlaBlaInstall extends Install Script
{
    public function __invoke($params = [])
    {
        $setActionAccessPermissions = $this->propagate(new SetActionAccessPermissions());
        $setActionAccessPermissions([
            '--' . SetActionAccessPermissions::OPTION_PERMISSIONS, [
                taoItems_actions_Items::class => [
                    'editClassLabel' => [
                        TaoItemsRoles::ITEM_CLASS_NAVIGATOR => ActionAccessControl::READ,
                        TaoItemsRoles::ITEM_CLASS_EDITOR => ActionAccessControl::WRITE,
                    ],
                ],
            ],
        ]);
    }
}
```